controller/config.py

```diff
@@ -27,6 +27,7 @@
     {"name": "configmaps", "group": ""},
     {"name": "services", "group": ""},
     {"name": "persistentvolumeclaims", "group": ""},
+    {"name": "persistentvolumes", "group": ""},
```

Comment:
I dont think we should merge this in and release. Lets wait until we have our own version of the controller that does not need access to PVs. I thought that was the plan?

Reply:
:+1: 
yes that's why it's still a draft